src/main/webapp/dev/js/common/instructorFeedbackResults.es6

```diff
@@ -266,7 +266,7 @@ function prepareInstructorFeedbackResultsPage() {
     const participantPanelType = 'div.panel.panel-primary,div.panel.panel-default';
 
     $('a[id^="collapse-panels-button-section-"]').on('click', (e) => {
-        const isGroupByTeam = document.getElementById('frgroupbyteam').checked;
+        const isGroupByTeam = $('#frgroupbyteam').get(0).checked;
```

Comment:
Use jquery functions: ``$('#frgroupbyteam').prop('checked');``. See if you can do the same for the rest of the code (unless the changes become too extensive).